tools/cell_census_builder/tests/test_util.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import pytest
+from scipy.sparse import coo_matrix, csr_matrix, triu
+
+from tools.cell_census_builder.util import array_chunker, is_nonnegative_integral, uricat
+
+
+def test_is_nonnegative_integral() -> None:
```

Comment:
need to add a few tests:
* empty (zero length) ndarray
* 2D ndarray (not just 1D)
* spmatrix of shape (0, 0)